public/model/layersFunctions.ts

```diff
@@ -61,3 +61,24 @@ export const referenceLayerTypeLookup = {
   [DASHBOARDS_MAPS_LAYER_TYPE.CUSTOM_MAP]: true,
   [DASHBOARDS_MAPS_LAYER_TYPE.DOCUMENTS]: false,
 };
+
+export const getDataLayers = (layers: MapLayerSpecification[]) => {
```

Comment:
In future we need to introduce a property to define whether a layer is data or base layer.

Reply:
+1